src/functions/Output.ps1

```diff
@@ -477,6 +477,83 @@ function ConvertTo-HumanTime {
     }
 }
 
+function Get-SkipRemainingOnFailurePlugin ($SkipRemainingOnFailure) {
```

Comment:
Move plugin to it's own file? Not related to screen output.

Reply:
Yeah wanted to do this, since it doesn't belong in `Output.ps1`. Will make sure to separate this into a separate script 👍